godot/entities/prefabs/npcs/fighter/fighter.tscn

```diff
@@ -22,7 +22,7 @@ unique_name_in_owner = true
 sprite_frames = ExtResource("2_fi7hp")
 animation = &"idle"
 autoplay = "idle"
-frame_progress = 0.965745
+frame_progress = 0.927114
```

Comment:
Estos cambios en el `frame_progress` no tienen que ver con la tarea. El proyecto los cambia constantemente pero no deberías hacer `git add` de este archivo. Ver https://github.com/GameNova-Studio/path-to-the-throne/issues/170 y su solución https://github.com/GameNova-Studio/path-to-the-throne/pull/171 .